jwst/jump/jump_step.py

```diff
@@ -29,9 +29,19 @@ class JumpStep(Step):
         min_sat_area = float(default=1.0) # minimum required area for the central saturation of snowballs
         min_jump_area = float(default=5.0) # minimum area to trigger large events processing
         expand_factor = float(default=2.0) # The expansion factor for the enclosing circles or ellipses
-        use_ellipses = boolean(default=False) # Use an enclosing ellipse rather than a circle for MIRI showers
+        use_ellipses = boolean(default=False) # deprecated
         sat_required_snowball = boolean(default=True) # Require the center of snowballs to be saturated
-        expand_large_events = boolean(default=False) # must be True to trigger snowball and shower flagging
+        min_sat_radius_extend = float(default=2.5) # The min radius of the sat core to trigger the extension of the core
+        sat_expand = integer(default=2) Number of pixels to add to the radius of the saturated core of snowballs
+        expand_large_events = boolean(default=False) # Turns on Snowball detector for NIR detectors
+        find_showers = boolean(default=False) Turn on shower flagging for MIRI
+        edge_size = integer(default=25) # Size of region on the edges of NIR detectors where a sat core is not required
+        extend_snr_threshold = float(default=1.2) The SNR minimum for detection of extended showers in MIRI
+        extend_min_area = integer(default=90) Min area of emission after convolution for the detection of showers
+        extend_inner_radius = float(default=1) Inner radius of the Ring2DKernal used for convolution
```

Comment:
Same issue with spelling of `Ring2DKernel` (in this and other lines)

Reply:
fixed